docs/randomblind.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Randomblind issuance
+---
+
+<script type="text/x-mathjax-config">
+  MathJax.Hub.Config({
+    extensions: ["tex2jax.js"],
+    jax: ["input/TeX", "output/HTML-CSS"],
+    tex2jax: {
+      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
+      displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
+      processEscapes: true
+    },
+    "HTML-CSS": { fonts: ["TeX"] }
+  });
+</script>
+<script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js"></script>
+
+*Randomblind* issuance is a new IRMA feature introduced in
+[irmago v0.6.0](https://github.com/privacybydesign/irmago/releases/tag/v0.6.0)
+that can give the user extra privacy
+guarantees during issuance.  Specifically, if this mode of issuance is enabled
+for an attribute within a credential type, this attribute is guaranteed to 
+  1) be random and 
+  2) remain unknown to the issuer. 
+
+This page explains how to enable this feature and later how this guarantee is
+upheld cryptographically.  The use case that sparked this feature is online
+voting, where voting secrecy plays a vital role. Since it is easier to
+explain by example, we'll use this use case as a practical example.
+
+
+## API
+
+Randomblind issuance is enabled in the scheme by adding the `Randomblind` XML
+tag to an attribute type within the issue specification of a given credential.
+For more information about schemes, see [this
+page](https://irma.app/docs/schemes/).  In the example below we enable this for
+the second attribute in the credential. It is possible to tag as many
+attributes as randomblind as the idemix public key allows.
+
+``` xml
+<IssueSpecification version="4">
+  ...
+  <Attributes>
+    <Attribute id="...">
+      ...
+    </Attribute>
+    <Attribute id="..." randomblind="true">
+      ...
+    </Attribute>
+  ...
+  </Attributes>
+...
+</IssueSpecification>
+```
+
+This means that during issuance, the issuer must *not* pass attribute values for
+attributes that are tagged as randomblind.
+
+As a concrete example we use the "Demo Voting Card" credential type, see [this
+page in the attribute
+index](https://privacybydesign.foundation/attribute-index/en/irma-demo.stemmen.stempas.html#irma-demo.stemmen.stempas.election).
+Even though the credential contains five attributes, the issuer must only give
+four concrete values to construct the credential. A requestor can start an
+issuance session to issue such a credential using the following command:
```

Comment:
I would start this sentence with `For example,` to hint to the reader that `irma session` is not the only possible way to start randomblind issuance sessions.